pilot/cmd/pilot-discovery/main.go

```diff
@@ -152,6 +184,28 @@ func init() {
 	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableCaching, "discoveryCache", true,
 		"Enable caching discovery service responses")
 
+	// Chiron options
+	var chironServiceNames string
+	var chironServiceNamespaces string
+	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.Chiron.EnableChiron, "enableChiron", false,
+		"Enable Chiron")
+	// Specifies the file path to k8s CA certificate.
+	// The default value is configured based on https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/:
+	// the CA certificate bundle is automatically mounted into pods using the default
+	// service account at the path /var/run/secrets/kubernetes.io/serviceaccount/ca.crt.
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Chiron.K8sCaCertFile, "k8sCaCertFile", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+		"Specifies the file path to k8s CA certificate.")
+	discoveryCmd.PersistentFlags().StringVar(&chironServiceNames, "serviceNames", "istio-galley,istio-sidecar-injector",
```

Comment:
That doesn't seem like a good way to configure this - mesh config, with proper yaml. 

k8sCertificates:
  services: 
   - istio-galley.istio-system
  - istio-sidecar-injector.istio-system
  - istio-pilot.istio-system

Reply:
Done: The format of a service will be service-name.service-namespace, with '.' as the delimiter, which means the service-name and service-namespace should not contain '.'.